### DISCUSSION TITLE
Avira shows snapmod as malware

### DISCUSSION THREAD

**Discussion by NMBlanco** [rodit/SnapMod#302] [General]
![Screenshot_20230327-195009](https://user-images.githubusercontent.com/12681440/228026032-d82db925-7e36-4c3a-85e1-cc8e756bb492.png)
Anyone got an idea what that means?

**Comment by RevealedSoulEven** [comment]
Lol. Avira shows every xposed module as a malware.

If you think it's a malware, it's open source. Go through it and examine if you find any malware. Still if you don't believe, build the apk yourself and use. 
It will still show malware😂
